fe/fe-core/src/main/java/org/apache/doris/datasource/iceberg/IcebergExternalTable.java

```diff
@@ -87,7 +96,11 @@ public void setTable(Table table) {
 
     @Override
     public Optional<SchemaCacheValue> initSchema(SchemaCacheKey key) {
-        return IcebergUtils.loadSchemaCacheValue(catalog, dbName, name, ((IcebergSchemaCacheKey) key).getSchemaId());
+        boolean isView = isView();
+        String remoteDbName = getRemoteDbName();
```

Comment:
Why should we using remote name?
Is this a bug that unrelated to this iceberg view feature?

Reply:
can fix in other PR